samples/SqlServerCacheSample/project.json

```diff
@@ -0,0 +1,19 @@
+{
+    "commands": {
+        "install-sqlservercache": "Microsoft.Framework.Caching.SqlServer",
```

Comment:
`install`? What does this operation actually do? create tables?

Also, what happens if you run this twice? I assume it will fail?